src/modules/xcm/tokens/index.ts

```diff
@@ -165,6 +165,16 @@ export const xcmToken = {
       originChain: Chain.ASSET_HUB,
       minBridgeAmount: '1',
     },
+    {
+      symbol: 'PEN',
+      isNativeToken: true,
+      assetId: '18446744073709551634',
+      originAssetId: '0',
```

Comment:
Looking at the other definitions, I think this should rather be 
```suggestion
      originAssetId: 'PEN',
```
though I'm not 100% sure if this is important.